completions/_nvme

```diff
@@ -396,6 +396,20 @@ _nvme () {
 				_arguments '*:: :->subcmds'
 				_describe -t commands "nvme ocp set-telemetry-profile options" _ocp_set_telemetry_profile_feature
 				;;
+			(get-latency-monitor)
+				local _ocp_get_latency_monitor_feature
+				_ocp_get_latency_monitor_feature=(
+				/dev/nvme':supply a device to use (required)'
+				--sel=':select from 0 - current, 1 - default, 2 - saved, 3 - supported'
+				-S':alias to --sel'
+				--nsid=':valid, invalid and inactive nsid'
+				-n':alias to --nsid'
```

Comment:
```diff
- 				-n':alias to --nsid'
+ 				-n':alias to --namespace-id'
```
When change an implemetaion could you please fix all realted implementaions since you change only the code part metioned but remain related code needed to change.